lib/http_aws_sigv4.c

```diff
@@ -612,14 +609,19 @@ CURLcode Curl_output_aws_sigv4(struct Curl_easy *data, bool proxy)
                                "Credential=%s/%s, "
                                "SignedHeaders=%s, "
                                "Signature=%s\r\n"
-                               "%s\r\n"
+                               /*
+                                * date_header is added here, only if it wasn't
+                                * pass as header (using -H),
```

Comment:
The header is actually set with `CURLOPT_HTTPHEADER` since this is library code.